src/goGenerateTests.ts

```diff
@@ -101,7 +101,14 @@ export async function generateTestCurrentFunction(): Promise<boolean> {
 	if (funcName.includes('.')) {
 		funcName = funcName.split('.')[1];
 	}
-	return generateTests({ dir: editor.document.uri.fsPath, func: funcName }, getGoConfig(editor.document.uri));
+
+	const currentFilePath = editor.document.fileName;
+	if (!currentFilePath.endsWith('.go')) {
```

Comment:
This check is already done in the `checkActiveEditor()` helper function which is called a few lines above. So we dont need it here.